cms/envs/common.py

```diff
@@ -1245,7 +1246,7 @@
 
 ############## Settings for Studio Context Sensitive Help ##############
 
-DOC_LINK_BASE_URL = None
+HELP_TOKENS_INI_FILE = REPO_ROOT / "docs" / "cms_config.ini"
```

Comment:
Does there need to be a corresponding configuration change? I'd recommend having a devops reviewer on this to be sure.